### DISCUSSION TITLE
Installation

### DISCUSSION THREAD

**Discussion by LMBits** [painebenjamin/app.enfugue.ai#133] [Q&A] [answered]
Not sure exactly how I should do this but by simply running the command and selecting portable I get this.

![Snippet_004](https://github.com/painebenjamin/app.enfugue.ai/assets/33180658/093a32fc-1d57-4509-b784-2e518cf7b1d9)

I also tried cloning the repo just in case but that didn't help neither.

**Comment by LMBits** [comment, marked as answer]
All solved.

**Reply by zeflash01-sketch** [reply]
may you teach how to solve it please ?
